### ISSUE TITLE
Out of Date - Couchbase Transactions .Net Documentation

### ISSUE THREAD

**Issue by cameronabell** [couchbase/docs-sdk-dotnet#271]
My team is currently upgrading our projects to .Net 6.0. Some of them are referencing Couchbase.Transactions 1.1.0 which now appears to be [deprecated](https://github.com/couchbase/couchbase-transactions-dotnet). 

We are unable to build with the reference to Couchbase.Transactions as it is not compatible with .Net 6. We tried to remove the reference but cannot figure out how to get transactions to work using only the SDK. The [.Net documentation](https://docs.couchbase.com/dotnet-sdk/current/howtos/distributed-acid-transactions-from-the-sdk.html) still refers to using Couchbase.Transactions 1.1.0. I assume this documentation is out of date since the github shows it is depricated.

TL;DR: The github and the .Net documentation seems to have conflicting information. How should we proceed with upgrading to .Net 6.0 while continue using Couchbase transactions?

**Comment by RiPont** [comment]
Hi @cameronabell ,

Couchbase.Transactions is not deprecated, only the original repo.  The source code has been moved to the couchbase-net-client repo.

The NuGet package targets .NET Standard 2.1, which is still supported by .NET 6.0.

What sort of build errors are you getting? Moving to .NET 6 sometimes creates erroneous compatibility problems, depending on your build process.

For example, https://stackoverflow.com/questions/70250523/net-6-does-not-support-net-standard-2-1

**Comment by cameronabell** [comment]
That's the exact error we are getting. This is helpful. I will go ahead and close this for now. Thanks.
